pkg/env/certificateRefresher_test.go

```diff
@@ -265,8 +280,7 @@ func TestRefreshingCertificate(t *testing.T) {
 				return
 			}
 
-			time.Sleep(test.sleep)
-
+			start <- struct{}{}
```

Comment:
At this point initial fetch happened and follow ticks can proceed.

Reply:
I would add this as a code comment